lib/Dialect/FIRRTL/Transforms/ExpandWhens.cpp

```diff
@@ -258,7 +284,9 @@ class LastConnectResolver : public FIRRTLVisitor<ConcreteT> {
     return b.create<ConnectOp>(loc, dest, newValue);
   }
 
-  void visitDecl(WireOp op) { declareSinks(op.getResult(), Flow::Duplex); }
+  void visitDecl(WireOp op) {
+    declareSinks(op.getResult(), Flow::Duplex, /*local=*/true);
```

Comment:
Is there a test covering this? Little surprised it's required to initialize input ports of objects in wires, but not output ports.  Is that right?

Reply:
Thanks, you're right, this should be local=false. I'm adding a test now.